docs/user/plagiarism-check.rst

```diff
@@ -0,0 +1,162 @@
+.. _plagiarism-check:
+
+Plagiarism checks
+=================
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Plagiarism Check Overview
+--------------------------
+
+Artemis allows tutors and exercise instructors to check assignments submissions from students for plagiarism.
+With this feature, different types of assignments can be checked in Artemis, including programming assignments, modeling assignments, and text assignments.
+To perform the plagiarism check, the responsible tutors must initiate the checking process for a specific task. Below, we describe the plagiarism check workflows from the perspective of various Artemis users using a UML Activity Diagram.
+
+.. _settings:
+
+Settings
+^^^^^^^^
+|run-settings|
+
+1. Similarity Threshold in % (minimum value 0, maximum value 100).
+
+  Ignore comparisons whose similarity is below this threshold.
+  A similarity of 0% means that there is no overlap between two submissions, a similarity of 50% means that about
+  half of two submissions are identical.
+  A similarity of 100% means that two submissions are completely identical.
+
+2. Minimum Score in % (minimum value 0, maximum value 100).
+
+  Consider only submissions with a score greater than or equal to this value.
+
+3. Minimum Size.
+
+  a. Default value: Consider only submissions whose size is greater or equal to this value.
```

Comment:
Which size is meant here? Lines? File size?

Reply:
The description is provided by the Artemis here. This is the default value, like a place holder.